src/components/LearnMore.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import logopic from '../img/blank.png';
+
+export default function LearnMore() {
```

Comment:
The structure of this component is very similar to the other ones that are going to be on the home page. Something to think about for a later PR is how you might combine the two into a more generic `HomeSection` component that take in things like the title, image, body, and direction.

Reply:
ok ill consider how to get this done